code/Controllers/CMSSiteTreeFilter.php

```diff
@@ -113,8 +114,12 @@ public function getNumChildrenMethod()
         return $this->numChildrenMethod;
     }
 
+    /**
+     * @deprecated 5.4.0 Will be renamed to getRecordClasses()
+     */
     public function getPageClasses($page)
     {
+        Deprecation::noticeWithNoReplacment('5.4.0', 'Will be renamed to getRecordClasses()');
```

Comment:
Didn't add the new implementations of these two methods because they're defined by the interface.